.github/workflows/chromatic.yml

```diff
@@ -52,9 +52,6 @@ jobs:
 
       - uses: ./.github/actions/ci-setup
 
-      # only packages have to be built as they might be imported by Storybooks
-      - run: pnpm build-packages
```

Comment:
is this not needed as Storybooks rely on the source condition?

Reply:
yes